streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -364,7 +364,7 @@ abstract class DStream[T: ClassTag] (
    * Get the RDD corresponding to the given time; either retrieve it from cache
    * or compute-and-cache it.
    */
-  private[streaming] final def getOrCompute(time: Time): Option[RDD[T]] = {
+  final def getOrCompute(time: Time): Option[RDD[T]] = {
```

Comment:
as @ymahajan mentioned, this can also be accessed using helper, so change should be reverted